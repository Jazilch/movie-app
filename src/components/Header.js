'use es6';

import React from 'react';
import styled from 'styled-components';

const HeaderWrapper = styled.div`
  display: flex;
  justify-content: space-between;
  align-items: center;
  color: #fff;
`;

const Logo = styled.div`
  color: red;
  font-weight: 700;
  font-size: 42px;
  letter-spacing: 1px;
  font-family: 'Bangers', cursive;
`;

const Nav = styled.nav`
  margin: 0 10px;
  color: #fff;
`;

const NavItem = styled.a`
  margin: 0 10px;
  color: #fff;
  text-decoration: none;
  font-size: 20px;
`;

const Header = () => (
  <HeaderWrapper>
    <Logo>Movie App</Logo>
    <Nav>
      <NavItem href="">Movies</NavItem>
      <NavItem href="">Shows</NavItem>
    </Nav>
  </HeaderWrapper>
);

export default Header;
