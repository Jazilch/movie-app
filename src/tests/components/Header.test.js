/* eslint-disable no-undef */

'use es6';

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../../components/Header';

describe('Header component', () => {
  let header;

  beforeEach(() => {
    header = shallow(<Header />);
  });

  it('renders header correctly', () => {
    expect(toJson(header)).toMatchSnapshot();
  });
});
