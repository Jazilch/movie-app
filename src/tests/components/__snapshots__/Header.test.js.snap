// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders header correctly 1`] = `
<styled.div>
  <styled.span>
    Movie App
  </styled.span>
  <styled.nav>
    <styled.a
      href=""
    >
      Movies
    </styled.a>
    <styled.a
      href=""
    >
      Shows
    </styled.a>
  </styled.nav>
</styled.div>
`;
